### PR TITLE
adding effects to statusres bard songs

### PR DESCRIPTION
**Pull request by 59blargedy**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [ ] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

I wanted to pull the bard songs in (thank you snugglesffxi) for our server and noticed that the effect scripts were blank. I may be misunderstanding, but i think that means the songs can cast but just apply an effect that does nothing at the moment. i couldn't spot anywhere where the effects do anything, but if i just missed it, happy to close this out.  i looked and used madrigal as an example, to add mods to the effect scripts. 

note: tangentially related - while trying to look for other examples, it looks like haste samba has this same issue, but has an extra script that confused me. if someone can look at that ability/effect script (there's a haste_samba_haste effect script but it never is referred to by the ability script???) if i'm right, ill be happy to fix haste samba in another pr.

### CODE REVIEW

**Review by Xaver-DaRed — APPROVED** [review]

File: scripts/globals/effects/aubade.lua [hunk 1]
```diff
@@ -4,12 +4,16 @@
 local effect_object = {}
 
 effect_object.onEffectGain = function(target, effect)
+    target:addMod(xi.mod.SLEEPRES, effect:getPower())
+    target:addMod(xi.mod.LULLABYRES, effect:getPower())
```
Comment (TeoTwawki) [inline scripts/globals/effects/aubade.lua:8]: This reminds me of some similar/related problems that used to have a tracker issue but were closed without resolution several process ago (this is more of a reminder note so I will click resolve after posting)

https://github.com/DarkstarProject/darkstar/issues/2531

tl;dr we're going to need more code dealing with multiple forms of sleep and resistance. Barsleep currently does not check lullaby either, and I do not know if it should. I'd rather not solve all of these by having even more res mods for them and then checking them all.

I think lullaby is legit in that it shoudl have a separate mod, I recall something in game calling it out this way (something that affects lullaby differently form other sleeps) but I can't recall _where_ - for the others it boils down to light or dark tho.

Comment (Snugglesffxi) [reply scripts/globals/effects/aubade.lua:8]: yeah sleep is just using one effect regardless of light or dark

**Review by KnowOne134 — APPROVED** [review]
